tests/workflows/basic_input_node_multi_stop/workflow.py

```diff
@@ -0,0 +1,67 @@
+from vellum.workflows import BaseWorkflow
+from vellum.workflows.inputs.base import BaseInputs
+from vellum.workflows.nodes.bases import BaseNode
+from vellum.workflows.state import BaseState
+
+
+class Inputs(BaseInputs):
+    input_value: str
+
+
+class State(BaseState):
+    state_value: str
+
+
+class StartNode(BaseNode):
+    input_value = Inputs.input_value
+    state_value = State.state_value
+
+    class Outputs(BaseNode.Outputs):
+        input_value: str
+        state_value: str
+
+    def run(self) -> Outputs:
+        return self.Outputs(input_value=self.input_value, state_value=self.state_value)
+
+
+class MiddleNode(BaseNode):
+    class ExternalInputs(BaseNode.ExternalInputs):
+        message: str
+
+
+class TransitionNode(BaseNode):
+    start_input = StartNode.Outputs.input_value
+    middle_message = MiddleNode.ExternalInputs.message
+
+    class Outputs(BaseNode.Outputs):
+        result: str
+
+    def run(self) -> Outputs:
+        return self.Outputs(result=f"{self.start_input} {self.middle_message}")
+
+
+class MiddleNode2(BaseNode):
+    class ExternalInputs(BaseNode.ExternalInputs):
+        message: str
+
+
+class EndNode(BaseNode):
+    start_input = StartNode.Outputs.input_value
+    start_state = StartNode.Outputs.state_value
+    middle_message = MiddleNode.ExternalInputs.message
+    middle_message2 = MiddleNode2.ExternalInputs.message
+
+    class Outputs(BaseNode.Outputs):
+        final_value: str
+
+    def run(self) -> Outputs:
+        return self.Outputs(
+            final_value=f"{self.start_input} {self.middle_message} {self.middle_message2} {self.start_state}"
+        )
+
+
+class BasicInputNodeWorkflow(BaseWorkflow[Inputs, State]):
+    graph = StartNode >> MiddleNode >> TransitionNode >> MiddleNode2 >> EndNode
```

Comment:
[nit] I think we could simplify this workflow to just MiddleNode >> MiddleNode2 >> EndNode, to focus on the case we're testing. At that point, we could just call them `InputNode1` and `InputNode2`